lib/Immediate/Immediate.cpp

```diff
@@ -37,9 +37,12 @@
 #include "llvm/Transforms/IPO.h"
 #include "llvm/Transforms/IPO/PassManagerBuilder.h"
 #include "llvm/Support/Path.h"
-
+#if defined(_MSC_VER)
+#include "Windows.h"
+#define dlopen(x, y)  LoadLibrary(x)
```

Comment:
Instead of a macro, can we use a wrapper function with a conditional implementation?